docs/cluster_spec.md

```diff
@@ -131,6 +131,17 @@ spec:
 
 Will result in the flag `--runtime-config=batch/v2alpha1=true,apps/v1alpha1=true`. Note that `kube-apiserver` accepts `true` as a value for switch-like flags.
 
+### dnsController
+
+This block contains configuration for `DNS-Controller`.
+
+```yaml
+spec:
+  watchIngress: true
+```
+
+Default kops behavior is false. `watchIngress: true` uses the default _dns-controller_ behavior which is to watch the ingress controller for changes. Set this option at risk of interrupting Service updates in some cases.
```

Comment:
"Set this option at risk of interrupting Service updates in some cases.".
It would be nice to add more info about when this would/could happen.